models/balancer/arbitrum/balancer_v2_arbitrum_trades.sql

```diff
@@ -1,5 +1,6 @@
 {{ config(
     schema = 'balancer_v2_arbitrum',
+    tags = ['dunesql'],
     alias = alias('trades'),
     partition_by = ['block_date'],
```

Comment:
let's partition by `block_month` here, where you can introduce a new column after `block_date` in the model

Reply:
again, applies universally